### PR TITLE
add chris' sns apache response as a template

### PR DESCRIPTION
**Pull request by abizer**
not to make light of security vulnerabilities or ISP's
occasionally overzealous reporting of them, but this is a
pretty decent template for responding to these emails, which
I imagine we'll be getting several more of until buster is
released and the version # is incremented.

### CODE REVIEW

**Comment by matthew-mcallister** [comment]
Nice quick-merge

File: www/templates/security-apache [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Hi ISP,
+
+Thank you for the report. We believe this is a false positive.
+
+The host in question runs Debian stable ("stretch") and has installed
+the apache2 package at version 2.4.25-3+deb9u3.
```
Comment (kpengboy) [inline www/templates/security-apache]: > version 2.4.25-3+deb9u3

You should make this a variable

File: www/templates/security-apache [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Because the Debian security team backports patches to software without
+upgrading to a newer point release, the banner version says 2.4.25
+despite containing fixes for these CVEs.
+
+Here is documentation from Debian indicating that the version we are
+running contains patches for these two CVEs mentioned:
+
+https://security-tracker.debian.org/tracker/CVE-2017-9788
+https://security-tracker.debian.org/tracker/CVE-2017-9789
```
Comment (kpengboy) [inline www/templates/security-apache]: These URLs also need to be variables too or a static URL that doesn't get outdated

**Comment by kpengboy** [comment]
In the long term it would be best to add an autoresponder which verifies that Apache has been updated and automatically sends the response with the right version number (and optionally Debian security notice links)

**Comment by chriskuehl** [comment]
Autoresponding to SNS sounds like a bad idea, but something that parses out the CVEs (via regex or something) that they mention in their email and then looks up the corresponding DSAs and formats all of that in a message would be useful.

**Pull request merged**